droidlet/interpreter/interpreter.py

```diff
@@ -48,10 +48,10 @@ def __init__(
                 raise Exception("multiple logical forms associated to single interpreter memory")
             logical_form_memid = logical_form_memids[0]
 
-        if (
-            logical_form_memid != "NULL"
-        ):  # if it were "NULL", this is a dummy interpreter of some sort...
-            logical_form_mem = agent_memory.get_mem_by_id(logical_form_memid)
+        if logical_form_memid != "NULL":
+            # if it were "NULL", this is a dummy interpreter of some sort...
```

Comment:
and right now the source for this would just be the unit tests, yeah ? If so, can we add this as a comment ?